blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -5,7 +5,7 @@ import { combineReducers } from 'npm:redux';
  * one or more other reducers which take on responsibility for
  * a discrete part of the state tree.
  *
- * Use the "ember generate reducer" command to create additional
+ * Use the "ember generate reducer [name]" command to create additional
```

Comment:
@ksnyde, I agree. The functionality you are speaking of was suggested and implemented [here](https://github.com/toranb/ember-redux/pull/20/files#diff-d5d915533b340ef3b6bfbe8aacdaa6c2R14).